code/__defines/subsystems.dm

```diff
@@ -1,22 +1,7 @@
-//Timing subsystem
-//Don't run if there is an identical unique timer active
-//if the arguments to addtimer are the same as an existing timer, it doesn't create a new timer, and returns the id of the existing timer
-#define TIMER_UNIQUE		0x1
-//For unique timers: Replace the old timer rather then not start this one
-#define TIMER_OVERRIDE		0x2
```

Comment:
this seems like the place for these to be

Reply:
You tell me which of these two files you prefer to keep these defines.
![image](https://user-images.githubusercontent.com/49106866/120087728-6a9b2500-c0b8-11eb-93ac-ae67af3cdbcd.png)

Reply:
subsystems.dm